src/test/java/com/learning/browsers/ChromeDriverGenerator.java

```diff
@@ -2,26 +2,17 @@
 
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.chrome.ChromeOptions;
 
 public class ChromeDriverGenerator implements BrowserDriver {
 
   private final WebDriver driver;
-  public static final String CHROME_USER_DATA = "C://Users//Bartek//AppData//Local//Google//Chrome//User Data";
 
   public ChromeDriverGenerator() {
-    driver = new ChromeDriver(getChromeOptions());
+    System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
```

Comment:
Zamiast podawania ścieżki na sztywno stwórz plik README.md w projekcie i dodaj tam informację o tym, że chromedriver ma być dostępny z poziomu $PATH.
Ustawianie property i ustawianie ścieżki na sztywno jest niebezpieczne, bo nie jest przenaszalne na inne platformy. Ja pracując na Mac, musiałbym modyfikować kod, żeby móc go uruchomić.

Reply:
For some reason it does not work for edge browser.
I guess it might be connected with this issue : [https://github.com/sitespeedio/edgedriver/issues/1](url).
When running tests it manages to open browser and then after a while on inactivity throws exception. 
On Chrome drive it works like a charm.